packages/docs/src/root.tsx

```diff
@@ -55,10 +55,11 @@ export default component$(() => {
         <meta charset="utf-8" />
         <script dangerouslySetInnerHTML={uwu} />
         <RouterHead />
+
         <ServiceWorkerRegister />
 
         <script dangerouslySetInnerHTML={`(${collectSymbols})()`} />
-        <Insights publicApiKey={import.meta.env.PUBLIC_QWIK_INSIGHTS_KEY} />
+        {/* <Insights publicApiKey={import.meta.env.PUBLIC_QWIK_INSIGHTS_KEY} /> */}
```

Comment:
is this on purpose?

Reply:
Yes, I left a comment somewhere which I probably edited.. to explain it was logging lots of `error` in the network logs, even though everything still worked normally 🤷‍♂️ . I think we can investigate later.